macros/helpers/prime_line.cfg

```diff
@@ -20,6 +20,11 @@ gcode:
     
     {% set line_height = 0.6 %}
 
+    # some more Macro parameters after retrieving defaults
+    {% set prime_line_x = params.START_X|default(prime_line_x)|float %}
+    {% set prime_line_y = params.START_Y|default(prime_line_y)|float %}
+    {% set prime_line_direction = params.LINE_DIRECTION|printer["gcode_macro _USER_VARIABLES"].prime_line_direction|string|upper %}
```

Comment:
I think you miss the "default" keyword here and this line will not work

Reply:
completed :)